src/components/Search/Search.js

```diff
@@ -1,17 +1,40 @@
-import React from 'react'
-import './Search.css'
+import React, { useState, useEffect } from "react";
+import "./Search.css";
+
+const Search = ({ setDrinks, drinks }) => {
+  const [searchInput, setSearchInput] = useState("");
+  const [error, setError] = useState("");
+
+  const handleSearch = e => {
+    e.preventDefault();
+    {
+      const filtered = drinks.filter(drink =>
+        drink.strDrink.toLowerCase().includes(searchInput.toLowerCase()),
```

Comment:
Nice! Per our discussion, using .startsWith() instead of .includes() to filter the search query will return all drinks that start with the characters the user inputs. Love that you made this change while we were talking.